openslides/core/views.py

```diff
@@ -35,6 +36,14 @@ class IndexView(utils_views.View):
     to the custom staticfiles directory. See STATICFILES_DIRS in settings.py.
     """
 
+    @classmethod
```

Comment:
I would prefer method decorating or using the decorater in the urls.py. See https://docs.djangoproject.com/en/1.7/topics/class-based-views/intro/#decorating-the-class